daemon/SecretsImpl/secretsrequestprocessor.cpp

```diff
@@ -1017,8 +1027,13 @@ Daemon::ApiImpl::RequestProcessor::setCollectionSecretGetAuthenticationCode(
         ikdRequest.setOperation(InteractionParameters::StoreSecret);
         ikdRequest.setInputType(InteractionParameters::AlphaNumericInput);
         ikdRequest.setEchoMode(InteractionParameters::PasswordEchoOnEdit);
-        ikdRequest.setPromptText(tr("Enter the passphrase to unlock the collection %1 in order to store secret %2")
-                                 .arg(secret.identifier().collectionName(), secret.identifier().name()));
+        //: This will be displayed to the user, prompting them to enter the passphrase to unlock the collection in which a new secret will be stored.  %1 is the application name, %2 is the  secret name, %3 is the collection name, %4 is the plugin name.
+        //% "Application %1 wants to store a new secret named %2 into collection %3 in plugin %4.  Enter the passphrase to unlock the collection."
```

Comment:
In UI design "Enter the passphrase.." strings are shown on separate label, requiring two separate translation strings.

In Sailfish OS we normally talk about passwords and not passphrases.

Reply:
I'm not sure if the authentication dialogs Andrew has implemented support multiple separated labels - perhaps they do, but not sure.

I will change passphrase to password.